### PR TITLE
Make all rtcwake cmd can select the RTC_DEVICE_FILE from config file (New)

### PR DESCRIPTION
**Pull request by seankingyang**
## WARNING: This modifies com.canonical.certification::sru-server

Modify all jobs will call the rtcwake cmd and didn't support for user to select which rtc device they want to use.
In rtcwake default will use the `rtc0`, and this change will also make if the `RTC_DEVICE_FILE` didn't define in config will also use the `rtc0`.

## Documentation

[OEMQA-4726 Modify jobs' command will call the rtcwake can support user to select which RTC devcie](https://warthogs.atlassian.net/browse/OEMQA-4726)

### CODE REVIEW

**Review by zongminl — CHANGES_REQUESTED** [review]
I see you made changes to some config_var in checkbox-snap, but why the followings are not changed:

- common_series_classic
- series_uc16
- series_uc18
- series_uc20
- series_uc22

Also there's a typo to be fixed, see my inline comment

**Comment by LiaoU3** [comment]
I am wondering if there are multiple RTC like rtc1, how should we handle it?

File: providers/base/units/stress/boot.pxu [hunk 1]
```diff
@@ -70,10 +71,10 @@ unit: template
 template-resource: reboot-run-generator
 template-unit: job
 plugin: shell
-environ: STRESS_BOOT_WAKEUP_DELAY STRESS_BOOT_WAIT_DELAY
+environ: STRESS_BOOT_WAKEUP_DELAY STRESS_BOOT_WAIT_DELAY RTC_DEVICE_FILE
 command:
  sleep "${{STRESS_BOOT_WAIT_DELAY:-120}}"
- rtcwake --mode off -s "${{STRESS_BOOT_WAKEUP_DELAY:-120}}"
+ rtcwake -v -d "${{RTC_DEVICE_FILE:-rtc0}}"--mode off -s "${{STRESS_BOOT_WAKEUP_DELAY:-120}}"
```
[the remark this answers is not in the document]
Comment (seankingyang) [reply providers/base/units/stress/boot.pxu]: Thanks~~ fix it

[the remark this answers is not in the document]
Comment (seankingyang) [reply providers/base/units/stress/boot.pxu]: $ find . -name "config_vars"
./common_series_uc/config_vars
./series_uc24/config_vars
./series_classic22/config_vars
./series_classic16/config_vars
./series_classic18/config_vars
./series_classic24/config_vars
./series_classic20/config_vars
```

**Review by zongminl — APPROVED** [review]
LGTM

**Pull request merged**